search-app/project.clj

```diff
@@ -29,19 +29,21 @@
                  [org.apache.httpcomponents/httpclient "4.5.6"]
                  [org.clojure/clojure "1.10.0"]
                  [org.clojure/data.csv "0.1.4"]
+                 [org.clojure/math.numeric-tower "0.0.4"]
                  [org.clojure/tools.reader "1.3.2"]
                  [ring/ring-codec "1.1.1"]
                  [ring/ring-core "1.7.1"]
                  [ring/ring-json "0.4.0"]
                  [selmer "1.12.5"]
-                 [org.geotools/gt-shapefile "22.3"]
-                 [org.geotools/gt-geojsondatastore "22.3"]
+                 [org.geotools/gt-shapefile "23-SNAPSHOT"]
```

Comment:
is it alright to use a developmental version?

Reply:
I remember James saying that 23 is scheduled for release soon. As soon as it is available you should update to use that.

Reply:
It's scheduled for some time in March.